### ISSUE TITLE
[BITV] Modal dialog "Choose target folder" - When the dialog is open, it is still possible to navigate through the elements in the background using keyboard, which should be prevented.

### ISSUE THREAD

**Issue by Pytal** [nextcloud/server#36719]
- [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-89806fe784-v3-n6
(Duplicates are present)

</details>
